Allure.Net.Commons/Allure.Net.Commons.csproj

```diff
@@ -6,7 +6,7 @@
         <Authors>Alexander Bakanov, Nikolay Laptev</Authors>
```

Comment:
@delatrie add your name to the authors please — you made a great contribution to this project.

Reply:
Done!